pool/payment.go

```diff
@@ -14,34 +15,44 @@ import (
 	bolt "go.etcd.io/bbolt"
 )
 
-// Payment represents an outstanding payment for a pool account.
+// PaymentSource represents the payment's source of funds.
+type PaymentSource struct {
+	BlockHash string `json:"blockhash"`
+	Coinbase  string `json:"coinbase"`
+}
+
+// Payment represents value paid to a pool account or collected fees.
 type Payment struct {
 	Account           string         `json:"account"`
 	EstimatedMaturity uint32         `json:"estimatedmaturity"`
 	Height            uint32         `json:"height"`
+	Source            *PaymentSource `json:"source"`
```

Comment:
Might be worth a comment here to explain the Source fields can possibly be empty if the payment was created in an old version of the database.